var/spack/repos/builtin/packages/heffte/package.py

```diff
@@ -68,6 +74,8 @@ class Heffte(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("rocsparse@3.8:", when="+magma+rocm", type=("build", "run"))
     depends_on("hipblas@3.8:", when="+magma+rocm", type=("build", "run"))
     depends_on("hipsparse@3.8:", when="+magma+rocm", type=("build", "run"))
+    depends_on("intel-oneapi-mkl@2023.2.0:", when="+sycl", type=("build", "run"))
```

Comment:
Do you recall why you are using explicit versions for mkl and intel mpi? We would like to move it forward.

Reply:
What do you mean by "explicit version"?

The `:` imply this version of newer, same as the ROCm libraries above.

Reply:
Sorry. I did not notice the ":"